src/abstract-project-create-command.ts

```diff
@@ -0,0 +1,271 @@
+import type {
```

Comment:
I don't understand the point of this file

- This is used only once, why split the projects/create?
- Why is that file not in `commands/project`

Reply:
- It will be used in the new commands
- the commands folder is the one "scanned" by oclif, so the file would appear as an actual command